config/module.config.php

```diff
@@ -43,8 +43,8 @@
     ],
     'validators' => [
         'factories' => [
-            Validator\DbRecordExists::class => Validator\Db\RecordExistsFactory::class,
-            Validator\DbNoRecordExists::class => Validator\Db\NoRecordExistsFactory::class,
+            'ZF\ContentValidation\Validator\DbRecordExists' => Validator\Db\RecordExistsFactory::class,
+            'ZF\ContentValidation\Validator\DbNoRecordExists' => Validator\Db\NoRecordExistsFactory::class,
```

Comment:
Just FYI: PHP will resolve the `::class`-notated strings just fine, even if the class does not exist. Essentially, it checks against existing imports and namespace declarations in order to resolve to a name, but does not check to see if that name exists. As such, the above was never an issue.